docker/docker-entrypoint.sh

```diff
@@ -33,8 +33,8 @@ trap alldone exit
 # so we don't throw away the one-time enrollment token
 
 IDENTITIES_DIR="/ziti-edge-tunnel"
-if ! mountpoint "${IDENTITIES_DIR}" &>/dev/null; then
-    echo "ERROR: please run this container with a volume mounted on ${IDENTITIES_DIR}" >&2
+if ! [[ -d "${IDENTITIES_DIR}" ]]; then
```

Comment:
The `mountpoint` command was used for this test to try and protect users from storing their json files on an ephemeral volume. I think we are losing that safety net here, which is probably fine - but the comment above should be removed.

Reply:
Comment removed. Warning added.